gradle.properties

```diff
@@ -7,7 +7,7 @@ cordaProductVersion = 5.0.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 626
+cordaApiRevision = 627
```

Comment:
I think this PR doesn't includes API breaking changes.

Reply:
Without this change I can't actually pick up boot config parameter for crypto, which is in a sense a breaking change.

Reply:
I agree it's a breaking change because `runtime-os` uses this constant for when inserting config, so without a version change somebody would pick this up and it would break an already created DB.

If we're cleaning up, I wonder if an `insert` for this key is missing from `data/db-schema/src/main/resources/net/corda/db/schema/config/migration/config-creation-v1.0.xml`
In theory it shouldn't matter as we, currently, always require the DB to be "primed" with the master key, but it would be more consistent, and if we were to move to a model where the key is passed as boot configuration, then this wouldn't require a further DB change.

Reply:
Left open for

> If we're cleaning up, I wonder if an insert for this key is missing from data/db-schema/src/main/resources/net/corda/db/schema/config/migration/config-creation-v1.0.xml
In theory it shouldn't matter as we, currently, always require the DB to be "primed" with the master key, but it would be more consistent, and if we were to move to a model where the key is passed as boot configuration, then this wouldn't require a further DB change.

which feels like a separate change, but I agree with minimising API churn to may be good to combine.